oeis.py

```diff
@@ -8,11 +8,12 @@
 from typing import Collection, Dict, List, Callable
 import sys
 import os
-
```

Comment:
This blank line was mandatory to split between Python stdlib imports and PyPI installed imports.